nodejs/eks/nodes/nodegroup.ts

```diff
@@ -1789,6 +1790,16 @@ export type ManagedNodeGroupOptions = Omit<
      * - [Amazon EKS AMI - Nodeadm Configuration](https://awslabs.github.io/amazon-eks-ami/nodeadm/doc/api/).
      */
     nodeadmExtraOptions?: pulumi.Input<pulumi.Input<NodeadmOptions>[]>;
+
+    /**
+     * The availability zone for the placement group.
+     */
+    placementGroupAvailabilityZone?: pulumi.Input<string>;
```

Comment:
Is placementGroupAvailabilityZone useful separately from enableEfaSupport?

If not perhaps we could force it to be required at type level:

```
efaSupport: {
  enable: true,
  placementGroupAvailabilityZone: "us-west-1b"
}
```

Reply:
I thought about that, but decided against it because we might add placement group support separately in the future. Tying it directly to EFA might maneuver us in a bad spot for the future